.golangci.reference.yml

```diff
@@ -361,14 +361,31 @@ linters-settings:
     # Forbid the following identifiers (list of regexp).
     # Default: ["^(fmt\\.Print(|f|ln)|print|println)$"]
     forbid:
+      # Builtin function:
       - ^print.*$
-      - 'fmt\.Print.*'
-      # Optionally put comments at the end of the regex, surrounded by `(# )?`
-      # Escape any special characters.
+      # Optional message that gets included in error reports.
+      - pattern: ^fmt\.Print.*$
```

Comment:
I think we ended up calling this "p" at https://github.com/golangci/golangci-lint/pull/3612/files#diff-cb1fbdbfccd3fd0d8fe9d9006138fbcd1afcc3a701b37180c6fe50ed0c5c390fR332.

Reply:
True, and I did the same for `ForbidigoPattern` in `linter_settings.go`. I've fixed the reference file to match.

It's worth calling out that we can use `pattern` here if `p` is considered too brief: we can map that in `linter_settings.go`:
```
// ForbidigoPattern corresponds to forbidigo.pattern and adds
// mapstructure support. The YAML field names must match what
// forbidigo expects.
type ForbidigoPattern struct {
	// patternString gets populated when the config contains a string as
	// entry in ForbidigoSettings.Forbid[] because ForbidigoPattern
	// implements encoding.TextUnmarshaler and the reader uses the
	// mapstructure.TextUnmarshallerHookFunc as decoder hook.
	//
	// If the entry is a map, then the other fields are set as usual
	// by mapstructure.
	patternString string

	Pattern string `yaml:"p" mapstructure:"pattern"` // <=====
	Package string `yaml:"pkg,omitempty" mapstructure:"pkg,omitempty"`
	Msg     string `yaml:"msg,omitempty" mapstructure:"msg,omitempty"`
}
```

I myself think golangci-lint should follow the choice made for upstream forbidigo and use `p`, too.